socorro/processor/breakpad_transform_rules.py

```diff
@@ -119,11 +119,6 @@ def _interpret_external_command_output(self, fp, processor_meta):
             return {}
 
     def _execute_external_process(self, command_line, processor_meta):
-        if self.config.get('chatty', False):
-            self.config.logger.debug(
-                "External Command: %s",
-                command_line
-            )
```

Comment:
I know I've used this a few times to debug configuration with mdsw in the current and new infrastructures. I think I'm ok with ditching it now and if we need something like it again in the future, we can implement something different.

Reply:
open to adding it forever also

Reply:
The cases where I needed it are all in the past, so I think let's ditch it for now.